### PR TITLE
[MARLIN-382] dashboard widget KPI creation

### PR DESCRIPTION
**Pull request by iseessel**
Along with https://github.com/maestrano/mnoe-admin-panel/pull/343.

1. Create endpoint to see dashboards. 
2. Used currently on frontend to view a user's dashboards. 
3. Proper authorizations set up for support users.

@ouranos @yuritomanek @alexmaestrano plz review.

### CODE REVIEW

**Comment by iseessel** [comment]
screen shot 2018-10-17 at 5 11 24 pm

File: api/app/controllers/mno_enterprise/jpi/v1/admin/impac/dashboards_controller.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+module MnoEnterprise
+  class Jpi::V1::Admin::Impac::DashboardsController < Jpi::V1::Admin::BaseResourceController
+    skip_before_action :block_support_users, only: [:index]
+    before_filter :authorize_support, only: [:index]
```
Comment (alexmaestrano) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/impac/dashboards_controller.rb]: Good guards

File: api/app/controllers/mno_enterprise/jpi/v1/admin/impac/dashboards_controller.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+      @dashboards = query.to_a
+    end
+
+    private
+
+    def authorize_support
+      return unless current_user.support?
+
+      case params.dig('where', 'owner_type')
```
Comment (alexmaestrano) [inline api/app/controllers/mno_enterprise/jpi/v1/admin/impac/dashboards_controller.rb]: 💥

**Comment by ouranos** [comment]
This should have been open on `4.0` (this backward compatible change on jpiv1)
The frontend is also on the `2.0` branch of the admin panel which has been depending on `4.0` so far.
Resetting 5.0 and re-merging

**Pull request merged**